.github/workflows/release-wasm-bindings-to-npm.yml

```diff
@@ -32,19 +34,19 @@ jobs:
 
       - name: Install Yarn
         run: npm i -g yarn
-        working-directory: client/bindings/wasm
+        working-directory: ${{ env.working-directory }}
```

Comment:
Just checking, is the underscore automatically transformed into a dash ?

Reply:
Probably not, messed up